src/translate.zig

```diff
@@ -153,6 +159,26 @@ pub fn bytes_from_object(env: c.napi_env, object: c.napi_value, comptime length:
     return result;
 }
 
+pub fn bytes_from_buffer(
+    env: c.napi_env,
+    buffer: c.napi_value,
+    output: []u8,
+    comptime key: [:0]const u8,
+) !usize {
+    const data = try slice_from_value(env, buffer, key);
+    if (data.len < 1) {
+        return throw(env, key ++ " must not be empty.");
+    }
```

Comment:
We could add a similar check to `client.zig` to require that a request body must be provided (at least for now), e.g. `body_size > 0`. Later we can allow some operations to not specify a body.

Reply:
@jorangreef  I'll make a change in the `tigerbeetle` repo and update the upstream in another PR.